### PR TITLE
Only one CR sample per CRD type in CSV alm samples

### PR DESCRIPTION
**Pull request by eguzki**
https://issues.redhat.com/browse/THREESCALE-7180

### CODE REVIEW

File: doc/operator-user-guide.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 * [3scale installation Backup and Restore using the operator (in *TechPreview*)](operator-backup-and-restore.md)
 * [Application Capabilities (in *TechPreview*)](operator-application-capabilities.md)
 * [APIManager CRD reference](apimanager-reference.md)
+    * [APIManager CR samples](cr_samples/apimanager/)
```
Comment (miguelsorianod) [inline doc/operator-user-guide.md:23]: We should include a link to the one from config/samples which is the simple one

**Review by miguelsorianod — CHANGES_REQUESTED** [review]
:+1: looks good. I left a minor comment

**Review by miguelsorianod — APPROVED** [review]
:+1:

File: doc/operator-user-guide.md (hunk 1 above)
Comment (eguzki) [reply doc/operator-user-guide.md]: Added

**Pull request merged**